orchestrator/src/androidtestorchestrator/parsing.py

```diff
@@ -383,13 +388,19 @@ def _process_set_property_cmd(self, cmd: str) -> Tuple[int, str]:
         key = items[-2]
         value = items[-1]
         if key == "locale":
-            self._service_app.device.set_locale(value)
```

Comment:
INFO ONLY:  We should probably look at capturing the changelocael apk as part of the distribution that gets installed to allow setting of locale.  This is not pallatable, but is used by many other mobile test companies including BitBar.  Source didn't seem to be available, and it uses some tricks to get access to potentially unstable/changeable interfaces internal to Android Java API.